src/Orleans/Serialization/SerializationTestEnvironment.cs

```diff
@@ -0,0 +1,34 @@
+namespace Orleans.Serialization
+{
+    using System.Collections.Generic;
+    using System.Reflection;
+
+    using Orleans.Runtime;
+
+    public class SerializationTestEnvironment
```

Comment:
I still dislike this, though - I will see if I can move it to a common test project.